custom/icds/messaging/indicators.py

```diff
@@ -121,6 +126,11 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    def get_messages(self, language_code=None):
+        return LSAggregatePerformanceIndicator(self.domain, self.supervisor).get_messages(language_code)
```

Comment:
Thanks. I initially misunderstood the spec. I filtered those here https://github.com/dimagi/commcare-hq/pull/15046/commits/aaafd1e844d9b10ce907d272ec361b2e3b6cc4f0